publish/dir.props

```diff
@@ -49,6 +49,5 @@
     <PublishRid Include="win-arm64" />
     <PublishRid Include="linux-arm" />
     <PublishRid Include="rhel.7-x64" />
```

Comment:
I think the reason we have non portable rids is to express the portable based installers for those platforms. So it does sort of track what we expect the output to be.